mifospay/src/main/res/layout/fragment_send.xml

```diff
@@ -18,7 +18,9 @@
         android:layout_marginBottom="0dp"
         android:orientation="vertical"
         tools:layout_editor_absoluteX="0dp"
-        tools:layout_editor_absoluteY="0dp">
+        tools:layout_editor_absoluteY="0dp"
+        android:layout_alignParentLeft="true"
```

Comment:
What are these changes for? I do not see any UI change after these.

Reply:
@luckyman20 To support older version than API 17 I made these changes.

Reply:
@shaliniv16 can you explain how this affects older api versions